.github/workflows/check.yaml

```diff
@@ -0,0 +1,29 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          cache: pnpm
```

Comment:
When using `pnpm` as package manager in a GH action you must install it explicitly.
Otherwise this command won't be found in the GH action environment.
https://github.com/zkitter/zkitter-js/actions/runs/4346096154/jobs/7591707849#step:3:15